JapaneseFilter/sections/adservers.txt

```diff
@@ -63,6 +63,7 @@
 ! Eimg.jp
 ||adingo.jp.eimg.jp^
 !
+||amoad.com^$third-party
```

Comment:
We have a similar rule in `AdGuard Mobile` filter. I will remove it after closing this pull request.

Reply:
Ah, sorry I forgot to enable Mobile filter on test.

Reply:
Yeah, it happens. Have you noticed the use of this server in countries other than Japan?

Reply:
I think only Japanese sites. but it's in my personal list for desktop, suggesting it's used on some desktop site. I can quickly search for.

Reply:
Here two examples:
```
https://www.carsensor.net/
https://retty.me/
```

Reply:
Ok, thanks. Then it will stay for Japan.